presto-main/src/main/java/com/facebook/presto/server/security/KerberosConfig.java

```diff
@@ -23,6 +25,7 @@
     private String serviceName;
     private File keytab;
 
+    @NotNull
```

Comment:
These changes are unrelated to the Hive change. Can you pull into a separate commit or PR?